apps/blocks.py

```diff
@@ -41,10 +41,22 @@ class Meta:
         icon = 'grip'
 
 
-class FaqBlock(blocks.StructBlock):
+class AccordionItemBlock(blocks.StructBlock):
     title = blocks.CharBlock()
     body = blocks.RichTextBlock(required=False)
 
+
+class FaqBlock(blocks.StructBlock):
+
+    accordion_items = blocks.ListBlock(AccordionItemBlock())
+
+    background_color = blocks.ChoiceBlock(required=False, choices=[
```

Comment:
Sorry, I wasn't more clear here earlier. I think you either need the `----` (the empty choice) or the white choice. If you go with the white, you should make this field requiered, so that the empty choice is gone. If you'd rather have the empty choice you could add a helptext like "Not choosing a colour will result in a FAQ block with the background of the page."